apps/list.c

```diff
@@ -958,6 +990,16 @@ static void list_type(FUNC_TYPE ft, int one)
     for (fp = functions; fp->name != NULL; fp++) {
         if (fp->type != ft)
             continue;
+	switch (ft) {
+	case FT_cipher:
+	    if (!is_cipher_available(fp->name)) continue;
```

Comment:
We break the continue onto the following line.

Reply:
Done. Thanks!